packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -186,6 +187,7 @@ function useBlockEditorSettings( settings, postType, postId, renderingMode ) {
 				userPatternCategories: getUserPatternCategories(),
 				restBlockPatternCategories: getBlockPatternCategories(),
 				sectionRootClientId: getSectionRootBlock(),
+				allowZoomLevel: true,
```

Comment:
I'm not sure what the criterion is to allow/disable here. Something about the presence of a `<main>` tag?

Reply:
We should not have an `allowZoomLevel`, instead if there is no `sectionRootClientId` means zoom out should not auto engage.